src/relite/DeliteBridge.scala

```diff
@@ -553,6 +559,7 @@ trait Eval extends OptiMLApplicationCompiler with StaticData {
           val first = firstVect.as[DenseVector[Double]]
           val second = secondVect.as[DenseVector[Double]]
           val res = first ** second
+          println("DIMENZIJE " + first.length() + " " + second.length())
```

Comment:
Printlns should not really go into the compiler. Something like logging is OK. Also, comments should be in English and pull requests should be reviewed.